src/lib/Behat/Page/ContentViewPage.php

```diff
@@ -109,12 +114,17 @@ public function __construct(
         $this->repository = $repository;
         $this->argumentParser = $argumentParser;
         $this->universalDiscoveryWidget = $universalDiscoveryWidget;
+        $this->ibexaDropdown = $ibexaDropdown;
     }
 
-    public function startCreatingContent(string $contentTypeName)
+    public function startCreatingContent(string $contentTypeName, string $language = null)
     {
         $this->contentActionsMenu->clickButton('Create content');
         $this->contentTypePicker->verifyIsLoaded();
+        if ($language !== null) {
```

Comment:
This code should still look like in v3.3:
```
        if ($language !== null) {
            $this->contentTypePicker->selectLanguage($language);
        }
```

IbexaDropdown should be a used in ContentTypePicker, not in ContentViewPage directly